.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+#### What does this PR do?
+
+[GPII-XXXX](https://issues.gpii.net/browse/GPII-XXXX)
+
+##### Description of the fix implemented
+
+##### Does this PR affect to the living environments? if so, how?
+
+<!--
+The following sections are optional, remove them if needed
+-->
+
+##### Notes for the reviewers
+
+##### Deployment plan
```

Comment:
I think there are cases where we don't want to expose too much of our infra internal mechanics, so it makes sense to keep deployment plan in the ticket.

Reply:
The issue tracker is also public. Although I agree that perhaps some privacy on this procedures would be good.

Reply:
You are right about Jira. I knew that all tickets are public by default, but my impression was that you needed to log in to be able to see ticket comments.

Reply:
I'm not worried about this info being public. IMO Github and Jira are bad choices for secrets regardless of the specifics :).

Tickets will often be the best place for a deploy plan if the feature involves multiple PRs. To take a recent example: in which of the ten PRs associated with https://issues.gpii.net/browse/GPII-3860 (or even the five PRs in that feature's [deploy plan](https://issues.gpii.net/browse/GPII-3860?focusedCommentId=39513&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-39513) would you expect to find "the" deploy plan? :)

That said, I think it's ok to leave this here since all of these template items are optional (as discussed above).

Reply:
> I'm not worried about this info being public. IMO Github and Jira are bad choices for secrets regardless of the specifics :).

👍 

I personally prefer GH for deploy plans, as Jira, it's formatting and comments are a pain to deal with, while GH with simple check-boxes that can be ticked as the deployment moves forward works better imho. Also everything is in one place when reviewing the PR.